input/pagecontent/bfdr-content-transitions.md

```diff
@@ -0,0 +1,94 @@
+|Number Other Outcomes |     motherPrenatal     |    -    | [ObservationNumberOtherPregnancyOutcomesVitalRecords] |       -       |    -     |
+|Date Last Other Outcome |     motherPrenatal     |    -    | [ObservationDateOfLastOtherPregnancyOutcome] |       -       |    -     |
+|Principal Source of Payment |    motherPrenatal     |    -    | [CoveragePrincipalPayerDelivery] |       -       |    -     |
+|Mother's Weight at Delivery |     motherPrenatal    |    -    | [ObservationMotherDeliveryWeightVitalRecords] |       -       |    -     |
+|Mother's Height |     motherPrenatal    |    -    | [ObservationMotherHeightVitalRecords] |       -       |    -     |
+|Mother's Prepregancy Weight|     motherPrenatal     |    -    | [ObservationMotherPrepregnancyWeightVitalRecords] |       -       |    -     |
+|Mother Received WIC Food|     motherPrenatal     |    -    | [ObservationMotherReceivedWICFood] |       -       |    -     |
+|Cigarette Smoking Before During Pregnancy|     motherPrenatal     |    -    | [ObservationCigaretteSmokingBeforeDuringPregnancy] |       -       |    -     |
+|Pregnancy Risk Factors |     medicalHealthInformation    |    -    | [ObservationPregnancyRiskFactorVitalRecords] |       -       |    -     |
+|Number Previous Cesareans |     medicalHealthInformation     |    -    | [ObservationNumberPreviousCesareansVitalRecords] |       -       |    -     |
+|Infections During Pregnancy |     medicalHealthInformation   |    -    | [ConditionInfectionPresentDuringPregnancy] |       -       |    -     |
+|Obstetric Procedures |     medicalHealthInformation    |    -    | [ProcedureObstetric] |       -       |    -     |
+|Characteristics Labor Delivery |     medicalHealthInformation   |    -    | [ProcedureInductionOfLabor] [ProcedureAugmentationOfLabor] [ObservationSteroidsFetalLungMaturation] [ObservationAntibioticsAdministeredDuringLabor] [ConditionChorioamnionitis] [ProcedureEpiduralOrSpinalAnesthesia] [ObservationNoneOfSpecifiedCharacteristicsOfLaborAndDelivery] |       -       |    -     |
+|Fetal Presentations |     medicalHealthInformation   |    -    | [ObservationFetalPresentation] |       -       |    -     |
+|Final Route Method Delivery|     medicalHealthInformation   |    -    | [ProcedureFinalRouteMethodDelivery] |       -       |    -     |
+|Maternal Morbidity |     medicalHealthInformation   |    -    |[ProcedureBloodTransfusion] [ConditionPerinealLaceration] [ConditionRupturedUterus] [ProcedureUnplannedHysterectomy] [ObservationICUAdmission] [ObservationNoneOfSpecifiedMaternalMorbidities]|       -       |    -     |
+|Birth Weight |     newbornInformation   |    -    | [ObservationBirthWeightVitalRecords] |       -       |    -     |
+|Gestational Age at Delivery |     newbornInformation    |    -    | [ObservationGestationalAgeAtDeliveryVitalRecords] |       -       |    -     |
+|Apgar Score |     newbornInformation    |    -    | [ObservationApgarScoreVitalRecords] |       -       |    -     |
```

Comment:
Flag in the comment column all of the profiles we propose to move from VRCL to BFDR....
We might want to duplicate them and call them WhateverTheyAreCalledNow_New just like we did for the profiles we moved from VRDR/BFDR/MDI into VRCL.   We will need to do that eventually when the sandbox gets split apart into the ballot content